src/util/file-filter.js

```diff
@@ -7,24 +7,22 @@ import {createLogger} from './logger';
 
 const log = createLogger(__filename);
 
-// Use this function to mimic path.resolve without resolving to absolute path.
-export const normalizeResolve = (file: string): string => {
-  // normalize
-  file = path.normalize(file);
-
-  // trim trailing slash
-  if (path.parse(file).base && file.endsWith(path.sep)) {
-    return file.slice(0, -1);
+// check if target is a sub directory of src
+export const isSubDir = (src: string, target: string): boolean => {
+  const relate = path.relative(src, target);
+  // same dir
+  if (!relate) {
+    return false;
```

Comment:
:+1: yep, I agree, I think that it is reasonable to check this condition in src/program.js (e.g. near here https://github.com/mozilla/web-ext/blob/master/src/program.js#L127 , or here where we raise other UsageError instances https://github.com/mozilla/web-ext/blob/master/src/program.js#L141-L152)